var.mk

```diff
@@ -14,14 +36,19 @@ PROGRAMATICA_SRC_FILE ?= \
 # The local file gets tried first, and if not usable the remote on gets fetched.
 # If both are unset or set to an empty string, programatica support is skipped.
 
-GHCVERSION := $(shell ghc --numeric-version)
-ifneq ($(findstring 7., $(GHCVERSION)),)
-GHC7RTSOPTS := -rtsopts
-endif
-
 OSNAME := $(shell uname -s)
 OSVERS := $(shell uname -v 2>/dev/null)
 
+ifeq "$(OSNAME)" "Darwin"
+    STACK_DEPENDENCIES_FLAGS := --flag gtk:have-quartz-gtk
+else
```

Comment:
delete else branch, since already initialized in the suggested change above. Also it would be better to move OSNAME, OSVERS setting lines to the top of the file and nest setting it into the  `ifneq ($(STACK),)` body.